go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/lws
 
-go 1.24.0
+go 1.23.0
```

Comment:
I agree with you. I think it is better to bump this to when the k8s dependencies are bumped to 1.33. But I'll defer the decision to owners.

Reply:
I am ok with aligning with the upstreaming k8s if it doesn't break anything